src/checkers/inference/model/serialization/ToStringSerializer.java

```diff
@@ -174,6 +175,27 @@ public String serialize(PreferenceConstraint preferenceConstraint) {
         return result;
     }
 
+    @Override
+    public String serialize(ArithmeticConstraint arithmeticConstraint) {
+        boolean prevShowVerboseVars = showVerboseVars;
+        showVerboseVars = false;
+
+        // format: result = ( leftOperand op rightOperand )
+        final StringBuilder sb = new StringBuilder();
+        sb.append(arithmeticConstraint.getResult().serialize(this));
+        sb.append(" = ( ");
```

Comment:
you can save the `sb` dereference 7 times. `StringBuilder#append()` method return the StringBuilder. So you can write this in a stream-like way. The same for the other `serailize` method below.